File: src/main/java/io/jsql/sql/parser/sqltest.java
package io.jsql.sql.parser;

import com.alibaba.druid.sql.ast.SQLStatement;
import com.alibaba.druid.sql.dialect.mysql.parser.MySqlStatementParser;

/**
 * Created by 长宏 on 2017/2/25 0025.
 * 测试
 */
public class sqltest {
    public static void main(String[] args) {
        String sql = "showdb tables;";
        MySqlStatementParser parser = new MySqlStatementParser(sql);
        SQLStatement mySqlStatement = parser.parseStatement();
        MSQLvisitor visitor = new MSQLvisitor();
        mySqlStatement.accept(visitor);

    }
}

File: src/main/java/io/jsql/sql/parser/sqltest.kt
/*
 * Java-based distributed database like Mysql
 */

package io.jsql.sql.parser

import com.alibaba.druid.sql.dialect.mysql.parser.MySqlStatementParser
import io.jsql.sql.ONullConnection

/**
 * Created by 长宏 on 2017/2/25 0025.
 * 测试
 */
object sqltest {
    @JvmStatic fun main(args: Array<String>) {
        val sql = "showdb tables;"
        val parser = MySqlStatementParser(sql)
        val mySqlStatement = parser.parseStatement()
        val visitor = MSQLvisitor(ONullConnection())
        mySqlStatement.accept(visitor)

    }
}
